src/OpenIDConnectClient.php

```diff
@@ -1027,8 +1027,8 @@ protected function verifyJWTclaims($claims, $accessToken = null) {
         return (($this->issuerValidator->__invoke($claims->iss))
             && (($claims->aud === $this->clientID) || in_array($this->clientID, $claims->aud, true))
             && (!isset($claims->nonce) || $claims->nonce === $this->getNonce())
-            && ( !isset($claims->exp) || ((gettype($claims->exp) === 'integer') && ($claims->exp >= time() - $this->leeway)))
-            && ( !isset($claims->nbf) || ((gettype($claims->nbf) === 'integer') && ($claims->nbf <= time() + $this->leeway)))
+            && ( !isset($claims->exp) || ((gettype($claims->exp) === 'integer' || gettype($claims->exp) === 'double') && ($claims->exp >= time() - $this->leeway)))
```

Comment:
It would be cleaner to use in_array for this. As example:

```php
(in_array(gettype($claims->exp), ['double', 'float', 'integer']) && ($claims->exp >= time() - $this->leeway)))
```